ts-defs/index.d.ts

```diff
@@ -0,0 +1,38 @@
+import { Accelerator, BrowserWindow, NativeImage, MenuItem, Menu } from "electron";
+
+declare module 'testcafe-browser-provider-electron' {
+    type DefinedMenuItemsActions =
```

Comment:
Could you please explain what is the purpose of this enumeration? AFAIK Electron apps don't have any predefined menu items.

Reply:
Common electron apps have this item and for better IDE supporting such as VSCode it would be the best option

Reply:
I've checked a similar example in VS Code, and looks like it ignores the string enum type and uses just string as a type name. Also, it doesn't suggest enum values as parameter values. 
##### x.ts
```typescript
type Enum = 'A' | 'B' | 'C';

export default function (arg: string [] | string | object | number | Enum) {
    
};
```

##### y.ts
```typescript
import x from './x';

x()
```

![image](https://user-images.githubusercontent.com/4479386/50287117-9be88300-0472-11e9-9e11-36677b91c215.png)

So I think this type is redudant here. You can always create a such type in your tests, or even create a new package that will contain common menu item names.